jax/_src/scipy/stats/_core.py

```diff
@@ -147,3 +147,15 @@ def rankdata(
   if method == "average":
     return .5 * (count[dense] + count[dense - 1] + 1).astype(dtypes.canonicalize_dtype(jnp.float_))
   raise ValueError(f"unknown method '{method}'")
+
+@_wraps(scipy.stats.sem, lax_description="""\
+Currently the only supported nan_policy is 'propagate'
```

Comment:
I think it would be fairly straightforward to also implement `omit` by calling `jnp.nanstd`, if you want to add that.